datastore/mongo/mongo_test.go

```diff
@@ -15,7 +16,9 @@ import (
 )
 
 func getDSN() string {
-	return os.Getenv("TEST_MONGO_DSN")
+	e := os.Getenv("TEST_MONGO_DSN")
+	fmt.Println(e)
```

Comment:
👀

Reply:
Yeah, I'm trying to understand why the tests are failing on CI but passing locally